client/locations-list-map.js

```diff
@@ -907,13 +1037,22 @@ function loadDataFile(url, dataToStore) {
   );
 }
 
+function getDatasetFilename(dataset) {
```

Comment:
This should take both dataset and countryCode

```javascript
if (dataset === 'makers') { return '/data-makers.json'; }
else { return `/data-${countryCode}.json`; }

Reply:
Done.